src/schemathesis/models.py

```diff
@@ -30,7 +30,7 @@ def empty_object() -> Dict[str, Any]:
 
 
 @attr.s(slots=True)
-class Endpoint:
+class Endpoint:  # pylint: disable=too-many-instance-attributes
```

Comment:
There is a pylint comment at the beginning of the file. Probably we need to remove it now